src/pkg/message/connect.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package message provides a rich set of functions for displaying messages to the user.
+package message
+
+import (
+	"fmt"
+
+	"github.com/zarf-dev/zarf/src/pkg/state"
+)
+
+// PrintConnectStringTable prints a table of connect strings.
+func PrintConnectStringTable(connectStrings state.ConnectStrings) {
```

Comment:
I think this is leftover from a merge. We moved the `PrintConnectStringTable` function to src/cmd